packages/babel-core/src/config/files/index-browser.ts

```diff
@@ -11,8 +11,7 @@ import type { CallerMetadata } from "../validation/options";
 
 export type { ConfigFile, IgnoreFile, RelativeConfig, FilePackageData };
 
-// eslint-disable-next-line require-yield
-export function* findConfigUpwards(
+export function findConfigUpwards(
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   rootDir: string,
 ): Handler<string | null> {
```

Comment:
Flow is failing because this should just return `string | null` now.

The other CI failure would be fixed by rebasing on `main`

Reply:
`yarn lint`, `yarn build`, and `yarn flow check` all passed locally, `eslint` suddenly found that `while (true)` (different rules in generator functions?). Mutter, mutter. Fixed.